content/docs/introduction/faq.md

```diff
@@ -89,6 +89,16 @@ Yes, with the experimental [Alertmanager](https://github.com/prometheus/alertman
 Yes, with [PromDash](/docs/visualization/promdash/) and [Console
 templates](/docs/visualization/consoles/).
 
+### Can I change the timezone? Why is everything in UTC?
+
+For monitoring purposes, it is absurd to use any timezone observing
```

Comment:
I already wrote that a million times more benign than I feel about this scourge of humanity. I will dilute it down another trillion times...